docs/asciidoc/virtual-graph.adoc

```diff
@@ -181,3 +185,64 @@ CALL apoc.graph.fromDocument("{'id': 1,'type': 'artist','name':'Genesis','member
 As a result we have a virtual graph with two nodes and one relationship:
 
 image::{img}/apoc.graph.fromDocument_1[width=800]
+
+
+In case this json:
+[source, json]
+----
+{
+    "id": 1,
+    "type": "Person",
+    "name": "Andrea",
+    "sizes": {
+        "weight": {
+            "value": 70,
+            "um": "Kg"
+        },
+        "height": {
+            "value": 174,
+            "um": "cm"
+        }
+    }
+}
+----
+You can manage the `sizes` property as value object so you manage it as follows:
+
+
+[source,cypher]
+----
+call apoc.graph.validateDocument(<json>, {valueObjects: {Person: 'sizes'}})
+----
+
+So the procedure will create a node with the following properties:
+[source, json]
+----
+{
+    "id": 1,
+    "type": "Person",
+    "name": "Andrea",
+    "sizes.weight.value": 70,
+    "sizes.weight.um": "Kg",
+    "sizes.height.value": 174,
+    "sizes.height.um": "cm"
+}
+----
+
+As specified you can also provide a set of value-object properties for a Label:
+
+[source,cypher]
+----
+call apoc.graph.validateDocument(<json>, {valueObjects: {Person: ['sizes', 'anotherPropertyKey']}})
```

Comment:
best use the config we talked about here, so we don't publish an API that we need to support and which is immediately deprecated

https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/1267#issuecomment-517628589